### ISSUE TITLE
Convert homepage fields to ACF 

### ISSUE THREAD

**Issue by advancedwpuser** [advancedwp/awp-2015#1]
The homepage is currently built with a relatively unknown Fields Generator. Convert the homepage to be fully ACF.

**Comment by kevinwhoffman** [comment]
On the topic of ACF front-end dependency (http://www.billerickson.net/advanced-custom-fields-frontend-dependency/) do we feel there is a need to use core functions over ACF functions as Erickson describes?

**Comment by advancedwpuser** [comment]
That'd be great. It's a minor point, but it sets a good standard for our coding practices.

**Comment by kevinwhoffman** [comment]
I can take this. Plan is to install ACF Pro locally and create the fields, then generate a local JSON export of the fields. That JSON file can be committed to the repo and you can synchronize that file with the production site as a way of "importing" the ACF field groups.

Does this sound right @khromov? I think you described it to me this way once.

Reference:
http://www.advancedcustomfields.com/resources/local-json/
http://www.advancedcustomfields.com/resources/synchronized-json/

**Comment by khromov** [comment]
@kevinwhoffman Yup. Personally I'd like to see the fields in awp-functionality. Here's some sample code I wrote a while back for how to do that:
http://snippets.khromov.se/store-advanced-custom-field-local-json-in-a-plugin/

I'm not over the moon regarding using non-free plugins though, since it makes it impossible for some users to contribute. Consider a user doesn't have ACF PRO, now they can't even get the site up and working. 

There are free field suites available such as:
https://wordpress.org/plugins/cmb2/
https://github.com/wp-papi/papi

Also we can use ACF 4 as long as we don't use any of the premium plugins.
Licensing info for ACF:
http://www.advancedcustomfields.com/resources/including-acf-in-a-plugin-theme/

@mathetos Your thoughts?

**Comment by kevinwhoffman** [comment]
I guess we should make a decision on premium plugins as a whole. Currently we are slated to use ACF Pro, Gravity Forms, and possibly some Awesome Support premium addons.

**Comment by mathetos** [comment]
@khromov I understand your thoughts on the Pro plugin issue. I definitely don't want to go crazy with them. I think it's pretty safe to say that most devs who would contribute will have a license for GF and most likely ACF Pro as well. I don't really see a way around using ACF because the ideas I have for the internal pages really REQUIRES Flexible fields and coding that out would be really horrible and slow us down far too much.

The premium Add-ons for Awesome Support really are only bonus functionality for the Code Reviewers, they aren't necessary for contributing to the theme in any way.

All that said, if we were to go free only, and custom code only (which I'm really attracted to the idea as a whole), we'd need to replace the current Contribution Wizard with Caldera or Formidable (since we're also using that already for the Slack invite page, but it's conditional fields are wonky compared to GF) and then the internal pages would need to use Beaver Builder instead of ACF Flexible fields. So it's possible, it just slows us down a bit.

Let's continue the conversation in Slack for now.

Also for now, @kevinwhoffman I'd say move forward with the JSON fields in the functionality plugin as @khromov suggested.

**Comment by khromov** [comment]
:+1: Sounds good
